test/load.test.ts

```diff
@@ -974,3 +1027,60 @@ describe('#cleanCachedPackage', () => {
     expect(renameSpy.mock.calls.length).toBe(0);
   });
 });
+
+describe('#getLatestVersion', () => {
+  let axiosSpy: jest.SpyInstance;
+
+  beforeAll(() => {
+    axiosSpy = jest.spyOn(axios, 'get').mockImplementation((uri: string): any => {
```

Comment:
We have a little of both in SUSHI (using nock and spying on axios). I see that in FPL we don't use nock at all yet, but do have a couple existing spies on axios. So... I think this is probably OK, particularly since it _works_. IIRC, I switched to nock on some modules in SUSHI because the jest spy was not acting reliably in some circumstances.

If we think there is sufficient value to switch to nock in FPL, I'd suggest we consider it as a future task rather than doing it in this PR.